docs/.ruby-version

```diff
@@ -0,0 +1 @@
+3.2.0
```

Comment:
```suggestion
2.7.4
```

I think we should change this version to match the version according to the dependency versions [page](https://pages.github.com/versions/)

Reply:
I already tested with `2.7.4,` and it works fine for me locally